src/main/scala/io/opentargets/etl/backend/drug/Molecule.scala

```diff
@@ -70,7 +70,7 @@ object Molecule extends LazyLogging {
         col("molecule_chembl_id").as("id"),
         col("molecule_structures.canonical_smiles").as("canonicalSmiles"),
         col("molecule_structures.standard_inchi_key").as("inchiKey"),
-        coalesce(col("molecule_type"), lit("unknown")).as("drugType"),
+        when(col("molecule_type").isNull || col("molecule_type") === "Unknown", lit("unknown")).otherwise(col("molecule_type")).as("drugType"),
```

Comment:
If I'm reading this correctly, "Unknown" and null will now both be replaced with "unknown".

I just wanted to double check that the intention is to change the sentinel value to all lowercase (unknown instead of Unknown), despite "Unknown" being the sole value in the past (prior to https://github.com/opentargets/issues/issues/3354)?

Reply:
@dhimmel That's right. I prefer lower casing just to reduce case sensitivity issues.

Reply:
Hmm, backwards compatability and consistency (rest of `molecule_type` values using Title Case) seem like bigger considerations here. I also like all lowercase, but this is not a value set we're designing from scratch at this point.